internal/querycoord/cluster.go

```diff
@@ -259,7 +259,7 @@ func (c *queryNodeCluster) watchDmChannels(ctx context.Context, nodeID int64, in
 	if targetNode != nil {
 		err := targetNode.watchDmChannels(ctx, in)
 		if err != nil {
-			log.Debug("watchDmChannels: queryNode watch dm channel error", zap.String("error", err.Error()))
+			log.Warn("watchDmChannels: queryNode watch dm channel error", zap.String("error", err.Error()))
```

Comment:
Maybe we should just return error and handlt it in upper level func?

Reply:
Please refer to https://github.com/uber-go/guide/blob/master/style.md (Error Wrapping chapter) and https://dave.cheney.net/2016/04/27/dont-just-check-errors-handle-them-gracefully (Only handle errors once
 chapter). They provide nice suggestions.

Reply:
Same problem occurs in other functions.